eng/Version.Details.xml

```diff
@@ -14,15 +14,15 @@
     </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.NETCore.App" Version="3.0.0-preview4-27519-04">
+    <Dependency Name="Microsoft.NETCore.App" Version="3.0.0-preview4-27520-07">
```

Comment:
Why do we need an updated core-setup?

Reply:
because I did not want to wait for master to catch up. dotnet was crashing on arm and core-setup has critical fix. At this point failing IPv6 tests are last blocking part. I'll take it with @ulisesh. You can ignore this PR as long as no-merge is set.